gitlab/changelog.d/17420.added

```diff
@@ -0,0 +1 @@
+Adds additional Geo metrics. *Note: Some metrics are only available depending on your Gitlab versions*
```

Comment:
Hmm, the comment about versions makes me wonder: which ones? I think as a customer reading this (and that's the intended audience for changelogs these days), I'd be interested to know that these metrics become available with v17 of gitlab. I'd also be interested to know if there are exceptions...